lib/tokyo_api/identity.rb

```diff
@@ -9,50 +11,43 @@ def full_user(id)
     end
 
     def tokyo_identity_user_path(id, with_subscription_status: false, required_fields: nil, opt_in_public_ids: nil, minimum_consent_level: nil, encrypted: nil)
-      path = "/#{normalized_base_path}user/#{url_escape(id)}"
+      path = String.new("/#{normalized_base_path}user/#{url_escape(id)}")
 
       params = []
-      unless required_fields.nil?
-        params << required_fields_param(required_fields)
-      end
+      params << required_fields_param(required_fields) unless required_fields.nil?
 
       if with_subscription_status
         params << 'with_subscription_status=true'
-        additional_subscription_parameters = path_for_subscription_status_params(opt_in_public_ids, minimum_consent_level, encrypted)
-        unless additional_subscription_parameters.blank?
-          params << additional_subscription_parameters
-        end
+        additional_subscription_parameters = path_for_subscription_status_params(opt_in_public_ids,
+                                                                                 minimum_consent_level, encrypted)
+        params << additional_subscription_parameters unless additional_subscription_parameters.blank?
```

Comment:
Can/should this be
```suggestion
        params << additional_subscription_parameters if additional_subscription_parameters.present?
```

Reply:
Yes, under our "prefer positives vs. negatives" style hint